src/core/formatters.ts

```diff
@@ -60,7 +60,7 @@ export module Formatters {
    * @returns {Formatter} A formatter that displays exactly [precision] decimal places.
    */
   export function fixed(precision = 3) {
-    verifyPrecision(precision);
+    precision = verifyAndAdjustPrecision(precision);
```

Comment:
sounds a bit weird. `ensurePrecision` sounds like if we put a precision of (say) `2.3`, we want to ensure that the precision is that, which is not true. We are actually amending the precision.

However, this is dependent on my comment bellow. If we decide to just throw a warning, this can be renamed back to `verifyPrecision()`